config.go

```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/packer/common"
+	"github.com/hashicorp/packer/helper/communicator"
+	"github.com/hashicorp/packer/helper/config"
+	"github.com/hashicorp/packer/packer"
+	"github.com/hashicorp/packer/template/interpolate"
+	"strconv"
+)
+
+const Unspecified = -1
```

Comment:
if internal structures are moved into VM cloning steps, we don't need such extra conventions.